packages/core/core/src/services/document-service/utils/unidirectional-relations.ts

```diff
@@ -12,7 +12,11 @@ import { UID, Schema } from '@strapi/types';
  * @param oldEntries The old entries that are being overridden
  * @returns An array of relations that need to be updated with the join table reference.
  */
-const load = async (uid: UID.ContentType, oldEntries: { id: string; locale: string }[]) => {
+const load = async (
+  uid: UID.ContentType,
+  oldEntries: { id: string; locale: string }[],
+  draftEntries: { id: string; locale: string }[]
```

Comment:
I think we should either add this to the JSdoc types above or remove them entirely

Reply:
Yeah I'll redo the code once we validate the behavior ;)